packages/crypto/README.md

```diff
@@ -1,9 +1,37 @@
-# Quick Start
+# Crypto
 
-The crypto.ts file in the Lit SDK handles cryptographic operations, including the generation, import, encryption and decryption of keys, as well as management of cryptographic shares using BLS and ECDSA schemes.
+A comprehensive cryptographic toolkit for the Lit Protocol SDK that handles secure key operations, encryption/decryption, and advanced cryptographic schemes.
 
-### node.js / browser
+## Installation
 
-```
+```bash
 yarn add @lit-protocol/crypto
 ```
+
+## Quick Start
+
+```typescript
+import { generatePrivateKey, encryptWithSignature } from '@lit-protocol/crypto';
+
+// Generate a new private key
+const privateKey = await generatePrivateKey();
+
+// Encrypt data with signature
+const encryptedData = await encryptWithSignature(data, signature);
```

Comment:
Same as above. There is a `encrypt` function but not a `encryptWithSignature` one